deepspeech.cpp

```diff
@@ -21,17 +21,17 @@ extern "C" {
                 }
             }
 
-            unsigned int getModelBeamWidth()
+            unsigned int beamWidth()
```

Comment:
Let me know if you'd prefer that the names of the wrapper methods stay aligned with the deepspeech.h functions instead of the Go functions. I haven't used CGo enough to know what the best practices are here, and there needs to be a naming difference at some point; I'm just not sure where.

Reply:
In all honesty I'm not an expert on CGO either but my position on this is "get the cleanest and most idiomatic API in GO files, the rest is less important". Therefore I don't think those C name changes are neither great or bad, but they are consistent which is fine by me.